protocol-units/da-sequencer/config/src/lib.rs

```diff
@@ -48,10 +52,18 @@ env_default!(
                 .expect("Bad da sequencer listener address.")
 );
 
+env_default!(
+        default_movement_da_sequencer_block_production_interval_millisec,
+        "MOVEMENT_DA_BLOCK_PRODUCTION_INTERVAL_MILLISEC",
+        u64,
```

Comment:
These env names are becoming rather long. We can clean up later. We could remove `Movement` word also `Da` because it's implied from the config path and name already `DaSequencer`